tests/functional/tests/bundles/ovn-central.yaml

```diff
@@ -0,0 +1,22 @@
+variables:
+  openstack-origin: &openstack-origin distro
+
+local_overlay_enabled: False
+
+series: jammy
+
+applications:
+
+  vault:
+    charm: ch:vault
+    num_units: 1
+    channel: latest/edge
+
+  ovn-central:
+    charm: ch:juju-verify-ovn-central
```

Comment:
Can we add note, todo or create issue to replace this in feature. This can be easily missed.

Reply:
Sure. I'm trying to get the change merged into the mainline ovn-central charm asap. When that happens I'll update the test bundle

Reply:
I can see that you added the comment, but there is typo in url.

Reply:
typo should be fixed